pkg/manager/router/router_score.go

```diff
@@ -318,6 +375,32 @@ func (router *ScoreBasedRouter) OnBackendChanged(backends map[string]*BackendHea
 	if len(backends) > 0 {
 		router.updateServerVersion()
 	}
+	if router.enablePause && len(backends) > 0 {
+		pause := true
+		for be := router.backends.Front(); be != nil; be = be.Next() {
+			backend := be.Value
+			if backend.Healthy() {
+				pause = false
+				break
+			}
+		}
+		if pause {
+			router.logger.Info("last backend become unhealthy, notify connections to pause")
+			for be := router.backends.Front(); be != nil; be = be.Next() {
+				backend := be.Value
+				backend.connScore--
```

Comment:
`connScore` should be updated when pausing each connection.

Reply:
OK, I will update it.